devenv.bat

```diff
@@ -46,6 +53,8 @@ rem Enable .NET Core Profiling API
 SET CORECLR_ENABLE_PROFILING=1
 SET CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
 SET CORECLR_PROFILER_PATH=%~dp0\src\Datadog.Trace.ClrProfiler.Native\bin\%profiler_configuration%\%profiler_platform%\Datadog.Trace.ClrProfiler.Native.dll
+set DD_TRACE_CALLTARGET_ENABLED=0
```

Comment:
I believe you should remove that, calltarget is enabled by default now

Reply:
fixed in 37c70b05f1aa0be83b995eca5193d6865db4e0a7